csharp.test/TestRowOrientedParquetFile.cs

```diff
@@ -113,14 +113,12 @@ public static void TestEmptyRowGroup([Values(false, true)] bool closeBeforeDispo
         [Test]
         public static void TestWriterDoubleDispose()
         {
-            // ParquetRowWriter is not double-Dispose safe (Issue 64)
-            // https://github.com/G-Research/ParquetSharp/issues/64
-
             using var buffer = new ResizableBuffer();
             using var outputStream = new BufferOutputStream(buffer);
             using var writer = ParquetFile.CreateRowWriter<(int, double, DateTime)>(outputStream);
```

Comment:
I think this was already testing the double dispose due to the `using var` statement here, although the comment is quite misleading, it should probably say something like "Test that ParquetRowWriter is double-Dispose safe"

Reply:
See the commit where this test was added with the fix here: https://github.com/G-Research/ParquetSharp/commit/ce713348566d62c8959ba772531bfe84405e9424

Reply:
Yep of course it was 🤦‍♂️